doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -107,15 +107,15 @@ Traceability to Reference Model.
 
 | Ref # | sub-category | Description |  Traceability |
 |----|----------------|----------------------|-----------|
-| `req.vim.01` | General | The Architecture **must** allow infrastructure resource sharing. | |
-| `req.vim.02` | General | The Architecture **should** support deployment of OpenStack components in containers. | |
-| `req.vim.03` | General | The Architecture **must** allow VIM to discover and manage NFVI resources. | |
+| `req.vim.01` | General | The Architecture **must** allow infrastructure resource sharing. | [RA-1 3.2. "Consumable Infrastructure Resources and Services"](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter03.md#32-consumable-infrastructure-resources-and-services) |
+| `req.vim.02` | General | The Architecture **should** support deployment of OpenStack components in containers. | [RA-1 4.3.2. "Containerised OpenStack Services"](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter04.md#432-containerised-openstack-services) |
+| `req.vim.03` | General | The Architecture **must** allow VIM to discover and manage NFVI resources. | [RA-1 5.2.7. "Placement"](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter05.md#527-placement) |
 | `req.vim.04` | General | The Architecture **must** support Enhanced Platform Awareness (EPA) only for discovery of infrastructure resource capabilities.| |
-| `req.vim.05` | General | The Architecture **must** include image repository management. | |
-| `req.vim.06` | General | The Architecture **must** allow orchestration solutions to be integrated with VIM. | |
-| `req.vim.07` | General | The Architecture **must** support a multi-tenanted environment. | |
-| `req.vim.08` | General | The Architecture **must** support resource tagging. | |
-| `req.vim.09` | General | The Architecture **must** support horizontal scaling. | |
+| `req.vim.05` | General | The Architecture **must** include image repository management. | [RA-1 4.3.1.2. "Glance"](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter04.md#4312-glance) |
+| `req.vim.06` | General | The Architecture **must** allow orchestration solutions to be integrated with VIM. | **Orchestration of what** |
+| `req.vim.07` | General | The Architecture **must** support a multi-tenanted environment. | [RA-1 3.2.1. "Multi-Tenancy"](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/openstack/chapters/chapter03.md#321-multi-tenancy-execution-environment) |
```

Comment:
`req.inf.ntw.04` | Network | The Architecture **should** support service function chaining. | **Is this VNFM/NFVO requirement** |
SFC can be a requirement for OpenStack architecture.
We didn't address it, but it can be added to the backlog for next release

Reply:
| `req.vim.06` | General | The Architecture **must** allow orchestration solutions to be integrated with VIM. | **Orchestration of what** |
I suppose it's service orchestration (NFVO). NFVO integration is provided through OpenStack APis, we can refer to APIs chapter for that.

Reply:
@pgoyal01 issue #1055 raised for SFC (backlog)

Reply:
@karinesevilla RA-1 Ch08 has SFC as an item in need of developing a solution for. There was an issue and pr now closed. So do we need Issue #1055

Reply:
@msiadna `req.vim.07`changed to "The Architecture **must** support multi-tenancy. "

Reply:
Ok